assignments/callbacks.js

```diff
@@ -26,26 +26,59 @@ const items = ['Pencil', 'Notebook', 'yo-yo', 'Gum'];
 
 
 function getLength(arr, cb) {
-  // getLength passes the length of the array into the callback.
-  
+  return cb(items.length);
 }
+getLength(items,function(length){
+  console.log(length);
+});
+
 
 function last(arr, cb) {
   // last passes the last item of the array into the callback.
+  for(let i=0;i<items.length;i++){
+    return cb(items[3]);
+  }
+  
 }
+last(items,function(item){
+  console.log(item);
+});
+
 
 function sumNums(x, y, cb) {
   // sumNums adds two numbers (x, y) and passes the result to the callback.
+  return x+y;
 }
+sumNums(2,3,function(add){
+  console.log(add);
+});
 
 function multiplyNums(x, y, cb) {
   // multiplyNums multiplies two numbers and passes the result to the callback.
+  return x*y;
 }
+multiplyNums(5,4,function(multiply){
+  console.log(multiply);
```

Comment:
So that here, `multiply` will have the value `x * y` and the result of it will be logged in the console.